src/SDKs/ContainerRegistry/ContainerRegistry.Tests/ContainerRegistry.Tests.csproj

```diff
@@ -12,9 +12,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="[5.0.2-preview,6.0)" />
+    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="[6.0.0-preview,7.0)" />
```

Comment:
@djyou please add reference to the latest version of resource manager nuget package and re-record all your test using the latest version of resource manager.